City-of-Helsinki/hel-fi-drupal-grants
# Feature/au 722 kiitossivu ongelmat

# [AU-722](https://helsinkisolutionoffice.atlassian.net/browse/AU-722)


There was 2 issues on the completion page:
- status check did not work
- the submission date was not shown until the application is updated from ATV after integration step
- Kasko form had incorrect subvention values.


## What was done


* Status check was broken because missing data-attribute on the page. That was added / restored.
* Submission data is taken from submission object changed time if proper submission time is missing
* Kasko subventions were added.

## How to install

* Make sure your instance is up and running on correct branch.
  * `git checkout feature/AU-722-kiitossivu-ongelmat`
  * `make fresh`
* Run `make drush-cr`

## How to test


* [x] Go to KASKO avustus service page while logged in
* [x] create new application
make sure you have these as "Avustuslajit":
![image](https://user-images.githubusercontent.com/892920/222394088-21e61a77-c1c3-4b2f-b69a-787302d7b4ca.png)
* [x] go to edit & fill all fields & submit the form
* [x] see that submission date is present without update from ATV
* [ ] see that status check works correctly and status is updated on the page



[AU-722]: https://helsinkisolutionoffice.atlassian.net/browse/AU-722?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
